packages/amplify-category-analytics/package.json

```diff
@@ -14,6 +14,9 @@
     "amplify",
     "aws"
   ],
+  "publishConfig": {
```

Comment:
Do we want it public?

Reply:
It's already public here: https://www.npmjs.com/package/amplify-category-function. 
If we don't do this then when release it, the change wouldn't be reflected on npm

Reply:
yes, every package that is scoped must be declared public.
fwiw, we had a release fiasco in the past when new package got introduced without that config, see https://github.com/aws-amplify/amplify-cli/pull/11029 .